Add sample for AI extraction (#2)
ballerina-guides/etl-samples: reviewSummary/README.md

## Steps to run the sample

### 1. Configuring the Config.toml file
1. Create a `Config.toml` file in the root directory and add the below content.
```toml
openAIKey = "<OPEN_AI_KEY>"
```
2. Create an [OpenAI account](https://platform.openai.com) and obtain 

> an [API key](https://platform.openai.com/account/api-keys) and replace with `<OPEN_AI_KEY>`.

Reviewer: ```suggestion
an [API key](https://platform.openai.com/account/api-keys) and use it as `<OPEN_AI_KEY>` in the `Config.toml`.
```

## Steps to run the sample

### Obtaining an OpenAI key
1. Create an [OpenAI account](https://platform.openai.com).
2. Obtain your [API key](https://platform.openai.com/account/api-keys) and include it in the `Config.toml` file.

### Updating the Config.toml file
```
openAIKey = "<OPEN_AI_KEY>"
```

### Running the sample
Run the following command from the root directory of the sample.
```ballerina

> bal run

Reviewer 2: ```suggestion
$ bal run
```

```

### Send a request
```

> POST http://localhost:8080/api/reviews/summary

Reviewer: Let's have the actual curl here. Check other exammples as well

{
  "reviews": ["I recently ordered a laptop and it was in very good condition and it was delivered it on time.
    But there were some software glitches and I couldn't find enough instructions to troubleshoot them.
    Would be great if they provided more detailed manuals.",
    "The smartphone I ordered arrived with a damaged screen. They need to be careful when packaging.",

>     "Extrmely satisfied with my orders. Would suggest this place to anyone.",

Reviewer 2: ```suggestion
    "Extremely satisfied with my orders. Would suggest this place to anyone.",
```
